kayenta-web/config/kayenta.yml

```diff
@@ -115,6 +115,12 @@ kayenta:
 #    endpoint:
 #      baseUrl: http://remotejudge.example.com/path/to/service
 
+# Retry configuration for metrics fetching from metrics storage (see: MetricsRetryConfigurationProperties)
+  retry:
```

Comment:
Would prefer `kayenta.metrics.retry`, as you mentioned.

Reply:
fixed